### PR TITLE
Source: vege.f90 fix absence of spread in LS when there's no …

### PR DESCRIPTION
**Pull request by usfs-fds**
…wind

### CODE REVIEW

File: Source/vege.f90 [hunk 1]
```diff
@@ -389,6 +389,14 @@ SUBROUTINE LEVEL_SET_FIRESPREAD(T,DT,NM)
          PHI_W_Y = C_ROTH * ((3.281_EB * UMF_MAG)**B_ROTH) * (SF%VEG_LSET_BETA / BETA_OP_ROTH)**(-E_ROTH)
          PHI_W_Y = PHI_W_Y*UMF_Y/UMF_MAG
 
+         IF (UMF_MAG /= 0.0_EB) THEN
```
Comment (rmcdermo) [inline Source/vege.f90]: @ruddymell please use
```
ABS(UMF_MAG)<TWO_EPSILON_EB
```
Thanks!

Comment (rmcdermo) [reply Source/vege.f90]: Rather, >

Comment (rmcdermo) [reply Source/vege.f90]: You can just update your fork and your PR will be updated as long as we do not merge first.

Comment (ruddymell) [reply Source/vege.f90]: Thanks, I just updated the fork to have ABS(UMF_MAG) > TWO_EPSILON_EB as suggested

**Pull request merged**